solr/Dockerfile

```diff
@@ -1,3 +1,3 @@
 # syntax=docker/dockerfile:1
 FROM solr:latest
-COPY ./jts-core-1.15.0.jar /opt/solr-8.9.0/server/solr-webapp/webapp/WEB-INF/lib/
\ No newline at end of file
+COPY ./jts-core-1.15.0.jar /opt/solr-8.10.1/server/solr-webapp/webapp/WEB-INF/lib/
```

Comment:
This is kind of a pain -- I didn't see a way to get this directory out of an environment variable or something.  Until we figure this out, anytime they rev the solr docker image we'll need to update our stuff.  I suppose we could pin our stuff to a particular version of solr, too.  Something to figure out.